.github/workflows/unit-tests.yml

```
@@ -0,0 +1,77 @@
+name: Ci suite
+on: [push, pull_request]
+jobs:
+    symfony:
+        name: PHPUnit tests (PHP ${{ matrix.php-versions }})
+        runs-on: ubuntu-latest
+
+        # Docs: https://docs.github.com/en/actions/using-containerized-services
+        services:
+            postgres:
+                image: postgres:latest
```

```suggestion
                image: postgres:11.7-alpine
```

I think you want to test against what's in docker-compose - it's a little bit old though. Maybe there should be a matrix for postgres versions, too? 🙈

---

Yeah good one, added `'11.20-alpine', '14.8-alpine', '15.3-alpine'` some random versions